tratto/src/main/java/star/tratto/util/FileUtils.java

```diff
@@ -0,0 +1,287 @@
+     *
+     * @param path a path
+     * @throws Error if an error occurs while creating the directory
+     */
+    public static void createDirectories(Path path) {
+        try {
+            Files.createDirectories(path);
+        } catch (IOException e) {
+            throw new Error("Error when creating directory " + path, e);
+        }
+    }
+
+    /**
+     * Creates an empty file. Creates parent directories if necessary. If the
+     * file already exists, then this method does nothing.
+     *
+     * @param path a file
+     * @throws Error if an error occurs while creating the parent directories
+     * or new file
+     */
+    public static void createFile(Path path) {
+        try {
+            createDirectories(path.getParent());
+            if (!Files.exists(path)) {
+                Files.createFile(path);
+            }
+        } catch (IOException e) {
+            throw new Error("Error when creating file " + path, e);
+        }
+    }
+
+    /**
+     * Recursively deletes a directory and its contents. If the file does not
+     * exist, then this method does nothing.
+     *
+     * @param dirPath a directory
+     * @throws Error if an error occurs while traversing or deleting files
+     */
+    public static void deleteDirectory(Path dirPath) {
+        if (!Files.exists(dirPath)) {
+            return;
+        }
+        try (Stream<Path> walk = Files.walk(dirPath)) {
+            walk
+                    .filter(p -> !p.equals(dirPath))
+                    .forEach(p -> {
+                        if (Files.isDirectory(p)) {
+                            deleteDirectory(p);
+                        } else {
+                            try {
+                                Files.delete(p);
+                            } catch (IOException e) {
+                                throw new Error("Error when trying to delete the file " + p, e);
+                            }
+                        }
+                    });
+            // delete root directory last
+            Files.delete(dirPath);
+        } catch (IOException e) {
+            throw new Error("Error when trying to delete the directory " + dirPath, e);
+        }
+    }
+
+    /**
+     * Returns the path of the target file or directory when (hypothetically)
+     * moved from the source directory to the destination directory.
+     * NOTE: This method does NOT perform the move. This method is different
+     * from {@link Path#relativize(Path)}, which gives a path to the
+     * destination, relative to the source.
+     *
+     * @param source the source directory
+     * @param destination the destination directory
+     * @param target the target file in the source directory
+     * @return the relative path of target in the destination directory. For
+     * example, let
+     * <pre>
+     *     source = [...]/[source]
+     *     destination = [...]/[destination]
+     *     target = [...]/[source]/[...]/[fileName]
```

Comment:
These comments are a bit misleading, because some of the "..." are required to be identical to one another and some are not.  Please rewrite to remove all "...", or if some remain, add a comment stating explicitly that they have no constraints.

Also, the comment should indicate that all (some?) of the file names must be equally relative.  For example, I could not run this from my home directory and pass "myDir" as `sourceDir` and "/homes/gws/mernst/myDir/targetFile.txt" as `target`.